virttest/utils_conn.py

```diff
@@ -795,13 +795,16 @@ class TLSConnection(ConnectionBase):
     ca_cakey_path: CA certification path, sometimes need to reuse previous cert
     scp_new_cacert: copy new CA certification, default is to always copy
     restart_libvirtd: default is to restart libvirtd
+    name_dict: A dict for required file names in libvirt or qemu style
+    qemu_tls: True for qemu native TLS support
     """
     __slots__ = ('server_cn', 'client_cn', 'ca_cn', 'CERTTOOL', 'pki_CA_dir',
                  'libvirt_pki_dir', 'libvirt_pki_private_dir', 'client_hosts',
                  'server_libvirtdconf', 'server_syslibvirtd', 'auth_tls',
                  'tls_port', 'listen_addr', 'tls_allowed_dn_list',
                  'custom_pki_path', 'tls_verify_cert', 'tls_sanity_cert',
-                 'ca_cakey_path', 'scp_new_cacert', 'restart_libvirtd')
+                 'ca_cakey_path', 'scp_new_cacert', 'restart_libvirtd',
+                 'name_dict', 'qemu_tls')
```

Comment:
maybe rename name_dict to more meaningful name

Reply:
@chunfuwen , update the variable name to 'credential_name'